docs/fsharp/get-started/get-started-vscode.md

```diff
@@ -1,65 +1,28 @@
 ---
-title: Getting Started with F# in Visual Studio Code with Ionide
+title: Get Started with F# in Visual Studio Code
 description: Learn how to use F# with Visual Studio Code and the Ionide plugin suite.
 keywords: visual f#, f#, functional programming, .NET, Visual Studio Code, vscode, Ionide
 author: cartermp
 ms.author: phcart
-ms.date: 09/28/2016
+ms.date: 02/28/2018
 ms.topic: article
 ms.prod: .net
 ms.technology: devlang-fsharp
 ms.devlang: fsharp
 ms.assetid: 49775139-082e-442f-b5a2-dd402399b5d2
 ---
 
-# Getting Started with F# in Visual Studio Code with Ionide
+# Get Started with F# in Visual Studio Code
 
 You can write F# in [Visual Studio Code](https://code.visualstudio.com) with the [Ionide plugin](https://marketplace.visualstudio.com/items?itemName=Ionide.Ionide-fsharp), to get a great cross-platform, lightweight IDE experience with IntelliSense and basic code refactorings.  Visit [Ionide.io](https://ionide.io) to learn more about the plugin suite.
```

Comment:
not related to your changes but remove the extra space before Visit

Reply:
Actually, my next action after this PR is to update this article, so I'll take care of the space-y problems then.